docs/data-ai/capture-data/lorawan.md

```diff
@@ -0,0 +1,686 @@
+   If your SX1302- or SX1303-based peripheral lacks a dedicated model, choose `lorawan:sx1302-hat-generic`.
+1. Click **Add module**, and enter a name for your gateway.
+1. Click **Create**.
+1. Click **Save** in the top right to apply your changes.
+
+{{% /tab %}}
+{{% tab name="JSON Configuration" %}}
+
+In the `components` section of your machine configuration, add the following objects:
+
+```json
+{
+  "name": "<lorawan-gateway-board>",
+  "api": "rdk:component:board",
+  "model": "viam:raspberry-pi:<your-pi-model>",
+  "attributes": {}
+},
+{
+  "name": "<your-gateway-name>",
+  "api": "rdk:component:sensor",
+  "model": "viam:lorawan:<gateway-sensor-name>",
+  "attributes": {
+    "board": "<lorawan-gateway-board>",
+    "spi_bus": "<spi-bus-number>",
+    "region_code": "<region-code>"
+  }
+}
+```
+
+Choose an appropriate board model from the [board components registry](/operate/reference/components/board/).
+
+Choose an appropriate gateway model from the following options:
+
+- `viam:lorawan:sx1302-waveshare-hat`: Waveshare LoRaWAN SX1302 Gateway HAT
+- `viam:lorawan:sx1302-hat-generic`: generic model for all other peripherals built using the SX1302 or SX1303 chips
+- `viam:lorawan:rak7391`: RAK7391 WisGate Connect
+
+Configure attributes based on the tables below:
+
+{{% /tab %}}
+{{< /tabs >}}
+
+You must configure the following attributes for SX1302- and SX1303-based LoRaWAN gateways:
+
+- `board`: The name of the [board component](/operate/reference/components/board/) that the peripheral is connected to. Used for GPIO pin control.
+- `reset_pin`: GPIO pin used for peripheral reset. Not configurable for `sx1302-waveshare-hat`.
+
+For generic peripherals, you must also configure `spi_bus`, `power_en_pin`, and `path`. For a full list of attributes, see the [module README](https://github.com/viam-modules/lorawan?tab=readme-ov-file#configuration-for-lorawan-gateways).
+
+You must configure the following attributes for RAK7391 gateways:
+
+- `board`: The name of the [board component](/operate/reference/components/board/) that represents the Raspberry Pi Compute Module inside the RAK7391. Used for GPIO pin control.
+
+## Add a node
+
+Complete the following steps to configure your node:
+
+{{< tabs >}}
+{{% tab name="Builder" %}}
+
+1. Navigate to the **CONFIGURE** tab of your machine's page in the [Viam app](https://app.viam.com).
+1. Click the **+** icon next to your machine part in the left-hand menu and select **Component or service**.
+1. Select the `sensor` type, type `lorawan`, then select the `lorawan` model that matches the name of your node.
+   If the name of your node does not appear in the list, choose the generic `lorawan:node` option.
+1. Click **Add module**, and enter a name for your node.
+1. Click **Create** to add the model and module to your machine.
+1. Configure the model attributes as follows, according to the node type:
+1. Click **Save** in the top right to apply your changes.
```

Comment:
Sorry I may have made a mistake here in my suggestion in the last review.
```suggestion
1. Configure the model attributes as follows, according to the node type:
```
or something like this, or move "Click save..." to the end
```suggestion
1. Configure the model attributes as described below, according to the node type.
1. Click **Save** in the top right to apply your changes.
```